cinema/serializers.py

```diff
@@ -13,12 +13,77 @@ def create(self, validated_data):
         return Movie.objects.create(**validated_data)
 
     def update(self, instance, validated_data):
-        instance.title = validated_data.get("title", instance.title)
+        instance.title = validated_data.get(
+            "title",
+            instance.title
+        )
         instance.description = validated_data.get(
-            "description", instance.description
+            "description",
+            instance.description
+        )
+        instance.duration = validated_data.get(
+            "duration",
+            instance.duration
         )
-        instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    first_name = serializers.CharField(max_length=255)
+    last_name = serializers.CharField(max_length=255)
```

Comment:
And actor can not exists without first and last name(same for cinema hall)

Reply:
I suppose "required=True" is unnecessary statement cause of i didn't define blank and null values. Same for cinema hall